### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by pallavit**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -106,7 +106,7 @@
 /sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/       @miaojiang
 
 # PRLabel: %App Configuration
-/sdk/appconfiguration/                                             @nisha-bhatia @ShivangiReja @AlexanderSher @jsquire
+/sdk/appconfiguration/                                             @m-redding @ShivangiReja @AlexanderSher @jsquire
```
Comment (jsquire) [inline .github/CODEOWNERS:109]:
```suggestion
/sdk/appconfiguration/                                             @ShivangiReja @jsquire
```

Comment (jsquire) [reply .github/CODEOWNERS:109]: Let's leave just myself and Shivangi for now, please.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -483,9 +483,13 @@
 # ServiceLabel: %Monitor %Service Attention
 /sdk/monitor/                                                      @SameergMS @dadunl
 
+# PRLabel: %Monitor - Ingestion
+/sdk/monitor/Azure.Monitor.Ingestion/                              @nisha-bhatia @JoshLove-msft @pallavit
```
Comment (jsquire) [inline .github/CODEOWNERS:487]: To confirm, this is not intended as an indication that we're viewing Josh as an owner of this, right?

File: .github/CODEOWNERS (hunk 1 above)
Comment (pallavit) [reply .github/CODEOWNERS]: Done!

**Review by jsquire — APPROVED** [review]

**Pull request merged**